android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java

```diff
@@ -372,39 +365,56 @@ public View getFormDataView(String address) {
     @Override
     public JSONObject getObjectUsingAddress(String[] address, boolean popup) throws JSONException {
         if (address != null && address.length > 1) {
-            if (RuleConstant.RULES_ENGINE.equals(address[0])) {
+            if(RuleConstant.RULES_ENGINE.equals(address[0]) && !address[1].endsWith(".yml")){
```

Comment:
More of a clarification.

What does this `!address[1].endsWith(".yml")` check?

Reply:
Its supposed to check if its not a .yml file, since for rule-dynamic the file is read in the factory and packed as a jsonObject, so it should be easy to differentiate between rules-file and rules-dynamic addresses

Reply:
ok.